api/v1_coin.go

```diff
@@ -25,12 +25,13 @@ func (app *ApiServer) v1Coin(c *fiber.Ctx) error {
 			artist_coins.website,
 			artist_coins.created_at
 		FROM artist_coins
-		WHERE artist_coins.mint = @mint
+		WHERE artist_coins.mint = @input
```

Comment:
I don't love the overloading.
Could we consider making /:mint the default, but adding a /coins/ticker/:ticker or something like that? WET good imo.